### PR TITLE
Add biography markdown for Aya Elnakeb

### PR DESCRIPTION
**Pull request by ayaelnakeb**

### CODE REVIEW

**Comment by elbionredenica** [comment]
The content of this markdown looks empty. Did you intend that? @ayaelnakeb

File: 2024/fall/Scheffler-MW@11am/AyaElnakeb.md
Comment (uliana1one) [inline 2024/fall/Scheffler-MW@11am/AyaElnakeb.md:1]: The name doesn't follow the requested format

**Review by elbionredenica — CHANGES_REQUESTED** [review]
The content of this markdown looks empty. Did you intend that? @ayaelnakeb

**Review by elbionredenica — CHANGES_REQUESTED** [review]
Also, the file name doesn't follow the guide. It should be name-surname.md.

**Review by elbionredenica — APPROVED** [review]

**Pull request merged**